lib/node_modules/@stdlib/math/stats/anova1/README.md

```diff
@@ -0,0 +1,164 @@
+# One Way ANOVA
+
+> Perform a one-way analysis of variance.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var anova1 = require( '@stdlib/math/stats/anova1' );
+```
+
+### anova1( x, factor, opts )
+
+For an [array][mdn-array] or [typed array][mdn-typed-array] of numeric values `x` and an [array][mdn-array] of classifications `factor`, one-way anova is performed. The hypotheses are given as follows:
+
+<!-- <equation class="equation" label="eq:hypotheses" align="center" raw="\begin{align*} H_{0}:& \; \mu_{1} = \mu_{2} = \dots = \mu_{k} \\ H_{a}:& \; \text{at least one} \; \mu_{i} \; \text{not equal to the others} \end{align*}" alt="Hypotheses of ANOVA"> -->
+
+<!-- </equation> -->
+
+The function returns an object containing the treatment and error equared errors, degrees of freedom, mean squared errors and both the p-value and f score.
+
+```javascript
+var out;
+var x;
+var y;
+
+x = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 ];
+y = [ 'Treatent A', 'Treatment B', 'Treatment C', 'Control', 'Treatment A', 'Treatment B', 'Treatment C', 'Control', 'Treatment A', 'Treatment B', 'Treatment C', 'Control' ];
+
+// Making the object
+out = anova1(x, y);
+/*
```

Comment:
After the `/*`, please add a `returns`. Also, let's put quotation marks around all object keys. Since this helps for parsing purposes, we always include them, even though they are not required in JS. However, they are required when keys contains special characters such as whitespace, as is the case here for `Treatment A`, `Treatment B`, `Treatment C`. In such cases,  placing `'` around the keys is mandatory

Reply:
Resolved in all files.